DivisionPromise.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Division</title>
+        <script>
```

Comment:
The assignment looks good. checkout the previous assignment comments same applies here